README.md

```diff
@@ -1,118 +1,106 @@
+## Freestyle Jobs
 
-<img src="./pics/freestyle-build-env.png" align="center" alt="freestyle build env" style="width: 80%; height: auto"/><br>
+Freestyle jobs must opt into the `Folder Properties` build wrapper from
+the `Build Environment` section of their configuration page in order to
+be able to access these properties as they would any other environment
+variable.
 
-Only then will they inherit properties defined by their parent or ancestor folders —e.g. Running `echo $FOO` in a Shell build step:
+![](docs/images/folder-properties-freestyle-config.png)
 
-<img src="./pics/freestyle-example-1.png" align="center" alt="freestyle build env" style="width: 80%; height: auto"/><br>
+Only then will they inherit properties defined by their parent or ancestor folders —e.g. Running `echo $FOO` in a Shell build step :
+
+![](docs/images/freestyle-example-1.png)
 
 #### SCM Step in Freestyle Jobs
 
 Starting with version 1.2, Freestyle jobs can also use folder properties to **define SCM parameters** —e.g. By defining an `SCM_URL` property pointing to the Git repository and a `BRANCH_SELECTOR` property pointing to the branch, tag or commit to be checked out:
 
-<img src="./pics/freestyle-example-scm-1.png" align="center" alt="freestyle build env" style="width: 80%; height: auto"/><br>
+![](docs/images/freestyle-example-scm-1.png)
 
-Then, descendant freestyle jobs can use that either as `$SCM_URL` and `$BRANCH_SELECTOR`:
+Then, descendant freestyle jobs can use that either as `$SCM_URL` and `$BRANCH_SELECTOR` :
 
-<img src="./pics/freestyle-example-scm-2.png" align="center" alt="freestyle build env" style="width: 80%; height: auto"/><br>
+![](docs/images/freestyle-example-scm-2.png)
 
- or `${SCM_URL}` and `${BRANCH_SELECTOR}`:
+ or as `${SCM_URL}` and `${BRANCH_SELECTOR}` :
 
-<img src="./pics/freestyle-example-scm-3.png" align="center" alt="freestyle build env" style="width: 80%; height: auto"/><br>
+![](docs/images/freestyle-example-scm-3.png)
 
-### Pipeline Jobs
+## Pipeline Jobs
 
-Pipeline jobs can use step `withFolderProperties` to access them either inside or outside a node step:
+Pipeline jobs can use step `withFolderProperties` to access them :
 
-    withFolderProperties{
-        echo("Foo: ${env.FOO}")
-    }
+**Using folder properties in a pipeline job**
 
-Jenkins deployments using some of the older versions of the
-[Structs Plugin](https://wiki.jenkins.io/display/JENKINS/Structs+plugin) will need to do this using the `wrap` meta-step.
-In such scenarios you should note that the wrapped syntax must run inside a node step.
+``` groovy
+withFolderProperties{
+    echo("Foo: ${env.FOO}")
+}
+```
 
-    node {
-        wrap([$class: 'ParentFolderBuildWrapper']) {
-            echo("Foo: ${env.FOO}")
-        }
-    }
+Jenkins deployments using some of the older versions of the [Structs
+Plugin](https://plugins.jenkins.io/structs/) will need
+to do this using the `wrap` meta-step :
 
-### Declarative Pipeline Jobs
```

Comment:
Shouldn't the declarative pipeline example be retained?  Or, maybe I missed something and it is now obsolete?